hack/azure-cluster-up.sh

```diff
@@ -63,8 +72,12 @@ echo -e "\nSpinning up Azure Kubernetes Cluster $CLUSTER_NAME"
 az aks create --resource-group $RESOURCE_GROUP --name $CLUSTER_NAME --enable-addons monitoring --generate-ssh-keys --service-principal $username --client-secret $password
```

Comment:
Can you also add the Service principal credentials caching?

Reply:
Will get to it once the remote installation is taken care of. Was originally going to do it in a seperate pr, but I could do it here as well.